twistr/events.go

```diff
@@ -571,23 +634,90 @@ func PlaySummit(s *State, player Aff) {
 	   Region (Europe, Asia, etc.) they Dominate or Control. The player with the
 	   highest modified die roll receives 2 VP and may degrade or improve the
 	   DEFCON level by 1 (do not reroll ties). */
+	regions := []Region{CentralAmerica, SouthAmerica, Europe, MiddleEast, Africa, Asia}
+	playerRoll := SelectRoll(s)
+	oppRoll := SelectRoll(s)
+	for _, region := range regions {
+		sr := ScoreRegion(s, region)
+		if sr.Levels[player] == Domination || sr.Levels[player] == Control {
+			playerRoll++
+		}
+		if sr.Levels[player.Opp()] == Domination || sr.Levels[player.Opp()] == Control {
+			oppRoll++
+		}
+	}
+	if playerRoll == oppRoll {
```

Comment:
note: need messaging

Also, I'd prefer this to be combined with the following if/else into a switch, for readability.